redhat-developer/red-hat-developers-documentation-rhdh
# RHIDP-2824 : chore(dynamic-plugins) generate dynamic...

### What does this PR do?

chore(dynamic-plugins) generate dynamic plugins table - 1.2.x

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
* https://issues.redhat.com/browse/RHIDP-2519 
* https://issues.redhat.com/browse/RHIDP-2824
* https://issues.redhat.com/browse/RHIDP-2834
* https://issues.redhat.com/browse/RHIDP-2835
* https://issues.redhat.com/browse/RHIDP-2836
* https://issues.redhat.com/browse/RHIDP-2837
* https://issues.redhat.com/browse/RHIDP-2838
* https://issues.redhat.com/browse/RHIDP-2839
* https://issues.redhat.com/browse/RHIDP-2840

Summarizing the changes in this PR:

* 5 new plugins added https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/1.2.x/rel-notes-rhdh/#_new-plugins-included-in-red-hat-developer-hub-1-2
* 1 plugin removed (the dynamic plugins info plugin is not a dynamic plugin – it's a static plugins that informs a user about which dynamic plugins are present
* updated plugin versions from what was in 1.1.1 to the latest included in 1.2.0
* some minor reformatting to remove the description column (manually maintained) in favour of links to the package readmes on npmjs.com
* content is now generated via script using information from package.json and dynamic-plugins.default.json


### How to test this PR?
run the script to see the output. Or look at the changes to modules/dynamic-plugins/rhdh-supported-plugins.adoc in this PR!

```
./modules/dynamic-plugins/rhdh-supported-plugins.sh -b 1.2.x --clean
./build/scripts/build.sh; google-chrome titles-generated/main/plugin-rhdh/index.html
```


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
